canonical-websites/www.ubuntu.com
# Removed the /juju page

## Done

- removed juju from the navigation and the site
- updated all the redirects to https://jujucharms.com
- updated the link on the /openstack, /public-cloud, /server, /server/hyperscale and /telecommunications page2
- updated the copy docs
- deleted the juju directory and pages

## QA

- see that juju is NOT in the footer and that /juju redirects to https://jujucharms.com
- see the link on http://0.0.0.0:8001/desktop/developers
- see the link on http://0.0.0.0:8001/openstack/index
- see the link on http://0.0.0.0:8001/public-cloud/index
- see the link on http://0.0.0.0:8001/server/hyperscale
- see the link on http://0.0.0.0:8001/server/index
- see the link on http://0.0.0.0:8001/telecommunications

## Issue / Card

Partially fixes #3282
